var/spack/repos/builtin/packages/intel-tbb/package.py

```diff
@@ -10,7 +10,6 @@
 
 from spack import *
 
-
```

Comment:
This change is causing flake8 tests to fail:
```
  var/spack/repos/builtin/packages/intel-tbb/package.py:13: [E302] expected 2 blank lines, found 1
```

Reply:
Should be fixed now